### DISCUSSION TITLE
Version

### DISCUSSION THREAD

**Discussion by kensagle** [doitandbedone/ispyagentdvr-docker#171] [Q&A] [answered]
Wondering why version tag latest is 3.1.3 and 3.1.7 is out.  I would rather not hard code the version and use latest for watchtower to update.  Is there a valid reason latest is still version 3.1.3?

**Comment by kensagle** [comment, marked as answer]
Looks like 3.1.8 is now marked as latest and was auto upgraded on my system by watchtower.

Thanks for creating and maintaining this docker.  Great work!

**Comment by doitandbedone** [comment]
Somehow I missed your message but yes, release tags are created
automatically while latest tag may take a bit longer as they need approval.
They shouldn't be that far apart however.
